cmake-scripts/generated/asy-tests-list.cmake

```diff
@@ -0,0 +1,116 @@
+
+# This file is automatically generated. Do not modify manually.
+# This file is checked in as part of the repo. It is not meant to be ignored.
+#
+# If more tests are added, run scan-asy-tests-cmake.py to re-generate
```

Comment:
Non-blocking suggestion: Add a separate test that runs scan-asy-tests-cmake.py and compares its output against this file. That way, the github CI will fail if someone forgets to run the script when they should.

Reply:
Added [check-no-missing-asy-tests](https://github.com/vectorgraphics/asymptote/actions/runs/13890094498/job/38860477695?pr=536) step @charlesstaats